package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@digitalbazaar/ecdsa-xi-2023-cryptosuite",
+  "version": "1.0.2-0",
```

Set this so our tooling can do the initial release properly.
```suggestion
  "version": "0.0.1-0",
```